### DISCUSSION TITLE
Does it support Graph RAG, can it communicate with images, such as deploying multimodal models, and can it describe the content in images

### DISCUSSION THREAD

**Discussion by libai-lab** [open-webui/open-webui#3781] [General]
Does it support Graph RAG, can it communicate with images, such as deploying multimodal models, and can it describe the content in images

**Comment by AnonTester** [comment]
It can use multimodal vision models like llava and interact with images. 
Test using llava-llama3:8b-v1.1-q4_0
![image](https://github.com/open-webui/open-webui/assets/40003252/641b7194-12a0-4dd3-9d55-11b4a67d8979)

**Comment by Kingbadger3d** [comment]
I dont know if the RAG System is used with multimodal models, but you can use ollama and multimodel systems like LLava to do what your asking.
